db/migrate/20230329132744_create_saved_forms.rb

```diff
@@ -0,0 +1,19 @@
+class CreateSavedForms < ActiveRecord::Migration[6.1]
+  def change
+    create_table :saved_forms, id: :uuid do |t|
+      t.text :email
+      t.text :secret_question
+      t.text :secret_answer
+      t.text :page_slug
+      t.text :service_slug
+      t.text :service_version
+      t.text :user_id
+      t.text :user_token
+      t.text :user_data_payload
```

Comment:
```suggestion
      t.string :email, null: false
      t.string :secret_question,  null: false
      t.string :secret_answer, null: false
      t.string :page_slug,  null: false
      t.string :service_slug,  null: false
      t.string:service_version,  null: false
      t.string :user_id.  null: false
      t.string :user_token,  null: false
      t.string :user_data_payload,  null: false
```
I think the char limit for `string` value is less than `text` - just me nitpicking so we ensure we do not save a ridiculously long bit of text in the db.

Also, adding the `null: false` as a precaution - I'm assuming we want all those attributes to never be null but obvs change as required.

Reply:
We'll need it to be `text` for the user_data_payload, the rest can probably be string. I purposely left them all nullable so that we can set them to null explicitly when we invalidate the record.